client/apollo/react/src/Pagination/PaginationCommon.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+  ComponentProps,
+  ComponentType,
+  ElementType,
+  type ComponentPropsWithoutRef,
+} from "react";
+import chevronBackward from "@material-symbols/svg-400/rounded/chevron_backward.svg";
+import chevronForward from "@material-symbols/svg-400/rounded/chevron_forward.svg";
+import {
+  ItemPagination,
+  type ItemPaginationProps,
+} from "./ItemPagination/ItemPaginationApollo";
+
+import { ELLIPSIS } from "./ItemPagination/ItemPaginationCommon";
+import { getClassName } from "../utilities/getClassName";
+
+import { ClickIcon } from "../ClickIcon/ClickIconApollo";
+
+type getItemsProps = {
+  numberPages: number;
+  onChangePage: (page: number) => void;
+  currentPage: number;
+  asItem?: React.ElementType;
```

Comment:
Je préfère importer uniquement les éléments dont j'ai besoin plutôt que d'appeler tout React pour lui demander un élément.

```suggestion
  asItem?: ElementType;
```

Reply:
en phase, ça doit être copilot qui me la autocomplété